### PR TITLE
Hw11

### PR DESCRIPTION
**Pull request by Artsapollo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>${jodatime.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <finalName>dateUtil</finalName>
```
Comment (un-install) [inline pom.xml]: WTF?

File: src/test/java/HW11/FruitShopUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+
+    @Test(expected = NullPointerException.class)
+    public void testAllReturnNothing() {
+        System.out.println("Test All Return Nothing");
+        fruitShop.allFresh(null);
+    }
+
+    @Test
+    public void testEmptyList() {
```
Comment (un-install) [inline src/test/java/HW11/FruitShopUnitTest.java]: The signatures is unclear

File: pom.xml (hunk 1 above)
Comment (un-install) [reply pom.xml]: this is very laud copy&paste

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>joda-time</groupId>
```
Comment (un-install) [inline pom.xml]: same

File: .idea/compiler.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SergeyPasko) [inline .idea/compiler.xml]: I dont like to see to idea-specific file. It must don't present in repo

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
```
Comment (SergeyPasko) [inline pom.xml]: Why?

File: src/main/java/HW11/Fruit.java [hunk 6]
```diff
@@ -0,0 +1,111 @@
+package HW11;
```
Comment (SergeyPasko) [inline src/main/java/HW11/Fruit.java]: One of the my taskswas - I would like to see at code that you wrote in home. Basic model must be in master, and in branch only changes

File: src/test/java/HW11/FruitShopUnitTest.java [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    @AfterClass
+    public static void afterClass() {
+        System.out.println("After Class");
+    }
+
+    @Test
+    public void testAllFresh() {
+        System.out.println("Test all fresh 1");
```
Comment (SergeyPasko) [inline src/test/java/HW11/FruitShopUnitTest.java]: It is don't need. Other places in tests also

File: src/test/java/HW11/FruitShopUnitTest.java [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+    public void testToLiveLong() {
+        System.out.println("Test to live long");
+        List<Fruit> ex = new ArrayList<>();
+        ex.add(orange);
+        List<Fruit> actual = fruitShop.checkByExpireDate(30);
+        Assert.assertEquals(ex, actual);
+    }
+
+    @Test
+    public void testFruitValidation() {
```
Comment (SergeyPasko) [inline src/test/java/HW11/FruitShopUnitTest.java]: bad method's names

File: src/test/java/HW11/FruitShopUnitTest.java [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+    public void testFruitsOfFruitType() {
+        System.out.println("Test fruit of fruit type");
+        List<Fruit> expect = new ArrayList<>();
+        expect.add(apple);
+        List<Fruit> actual = fruitShop.allFruitOfFruitType(FruitType.APPLE);
+        Assert.assertEquals(expect, actual);
+    }
+
+    @Test
+    public void testFruitsOfFruitType2() {
```
Comment (SergeyPasko) [inline src/test/java/HW11/FruitShopUnitTest.java]: What is reason? It is same to previous

**Pull request closed without merging**